ci/jenkins_tests/run_tune_tests.sh

```diff
@@ -162,6 +162,10 @@ $SUPPRESS_OUTPUT docker run --rm --shm-size=${SHM_SIZE} --memory=${MEMORY_SIZE}
     python /ray/python/ray/tune/examples/pbt_dcgan_mnist/pbt_dcgan_mnist.py \
     --smoke-test
 
+$SUPPRESS_OUTPUT docker run --rm --shm-size${SHM_SIZE} --memory=${MEMORY_SIZE} --memory-swap=-1 $DOCKER_SHA \
+    python /ray/python/ray/tune/examples/pbt_transformers/pbt_transformers.py \
+    --smoke-test
+
```

Comment:
btw this file is deprecated; you should use `tune/BUILD` instead. See https://github.com/ray-project/ray/pull/9609/files for the change

Reply:
added it to BUILD as well.